Makefile-test.am

```diff
@@ -57,7 +57,6 @@ if UNIT
 TESTS_UNIT  = \
     test/unit/CommonPreparePrologue \
     test/unit/CopyCommandHeader \
-    test/unit/GetNumHandles \
```

Comment:
This is an endless argument between devs I guess.
@flihp removed all statics, because unit tests. I did not care too much... ;-)

But I guess removing a unit test in favor of statics is not too good.
I guess we can remove the functions from the headers (I'm all for that) but then declare them locally in the corresponding test ?

Reply:
To declare private functions locally in test will not work if a function is static.
In the case of GetNumHandles we have a good coverage as it is called for every TPM command.

Reply:
Let's await the email outcome...

Reply:
Here is the coverage:
https://codecov.io/gh/tpm2-software/tpm2-tss/pull/1376/src/src/tss2-sys/sysapi_util.c?before=src/tss2-sys/sysapi_util.c#D1-286

Reply:
I know, but unit tests fail earlier, if you know what I mean... ;-)
OTOH, non-statics may lead to collisions...
So I'm undecided, but I guess we want to be consistent...